src/index.js

```diff
@@ -35,22 +35,36 @@ const cols = 64
  ***************************
  */
 
+// In terms of module structure I'd prefer to see the code related to
+// manipulating board state in one place and not here in the index.
 const wrappedIndex = (length, idx) => idx < 0 ? length + idx : (idx >= length ? idx - length : idx)
 const wrappedGet = (a, idx) => a[wrappedIndex(a.length, idx)]
 
+// Nit: It wasn't immediately obvious to me what r and c represent. x and y are
+// more common, otherwise I'd spell out row and col. Oh, and I still don't know
+// what a is. Array? It's a board in some functions and an array in others.
 export const get = (a, r, c) => wrappedGet(wrappedGet(a, r), c)
 
+// Nice. One thing that prettier would probably butcher :)
```

Comment:
I tried a couple different approaches with this, including slicing around the current cell and also building a 2d array of neighbor coordinates, but this approach turned out to be the fastest.